.github/workflows/sync-with-PRO.jsonnet

```diff
@@ -13,10 +15,63 @@ local gha = import 'libs/gha.libsonnet';
 
 local job = {
   'runs-on': 'ubuntu-latest',
-  steps: [
+  permissions: gha.write_permissions,
+  steps: semgrep.github_bot.get_token_steps + [
      {
-       run: 'echo TODO'
+      name: 'Checkout OSS',
+      uses: 'actions/checkout@v3',
+      with: {
+        ref: '${{ github.event.repository.default_branch }}',
+        // fetch all history
+        'fetch-depth': 0,
+        // Use the token provided by the JWT token getter above
+        token: semgrep.github_bot.token_ref,
+      },
      },
+     {
+      name: 'Checkout PRO',
+      uses: 'actions/checkout@v3',
+      with: {
+        repository: 'semgrep/semgrep-proprietary',
+        path: 'PRO',
+        token: semgrep.github_bot.token_ref,
+       },
+     },
+     {
+      name: 'Creating the branch and commiting to it',
+      env: {
+        BRANCHNAME: 'sync-with-PRO-x3-${{ github.run_id }}-${{ github.run_attempt }}',
```

Comment:
Out of curiosity, what `x3` means here ?

Reply:
I've removed it. It was because at some point the job was failing because the branch
was existing already and so I patched it.